html-extract.js

```diff
@@ -41,7 +42,7 @@ module.exports = function (opts) {
           self.push(new gutil.File({
             // Name: id or tag + index.
             path: file.path + "-" + (el.attribs.id || el.tagName + "-" + i),
-            contents: new Buffer(el.children[0].data)
+            contents: new Buffer(trim.left(el.children[0].data))
```

Comment:
Yeah to be accurate it'd be good if we knew the indentation level of the JS coming out of the script element. I didn't know a quick way to do that though. Does cheerio give indentation level?